provider/internal/builtin/service_client.go

```diff
@@ -61,7 +61,7 @@ func (p *builtintServiceClient) Evaluate(cap string, conditionInfo []byte) (prov
 				continue
 
 			}
-			ab, err := filepath.Abs(filepath.Join(p.config.Location, match))
+			ab, err := filepath.Abs(match)
```

Comment:
`walkDir` is already appending the base location to the matching filenames because we are passing it as root path. We never found this because location for builtin was empty. I still need to test this with shim once just to confirm it doesn't break anything else.

Reply:
looking good with the shim tests

```
Overall success rate: 80.75% (1909/2364)
```